<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>notepad</class>
 <widget class="QWidget" name="notepad">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>475</width>
    <height>592</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Speaker Recognition</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>451</width>
     <height>541</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="Enrollment">
    <attribute name="title">
     <string>Enrollment</string>
    </attribute>
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>0</y>
       <width>401</width>
       <height>201</height>
      </rect>
     </property>
     <property name="title">
      <string>User Info</string>
     </property>
     <widget class="QWidget" name="verticalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>40</y>
        <width>241</width>
        <height>121</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Name:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="Username">
           <property name="placeholderText">
            <string extracomment="Enter your name"/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Age:   </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="Userage">
           <property name="maximum">
            <number>100</number>
           </property>
           <property name="value">
            <number>20</number>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>Sex:   </string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="Usersex">
             <item>
              <property name="text">
               <string>Male</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Female</string>
              </property>
             </item>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="verticalLayoutWidget_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>131</width>
        <height>151</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="Userimage">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="UploadImage">
         <property name="text">
          <string>Upload Photo</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QComboBox" name="Userchooser">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>10</y>
        <width>171</width>
        <height>29</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>(Choose from list)</string>
       </property>
      </item>
     </widget>
     <widget class="QPushButton" name="UpdateInfo">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>160</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Update Info</string>
      </property>
     </widget>
     <widget class="QPushButton" name="ClearInfo">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>160</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Clear Info</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>200</y>
       <width>421</width>
       <height>291</height>
      </rect>
     </property>
     <property name="title">
      <string>Voice Enrollment</string>
     </property>
     <widget class="QTextBrowser" name="Textreading">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>60</y>
        <width>411</width>
        <height>101</height>
       </rect>
      </property>
      <property name="html">
       <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Cantarell'; font-size:11pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Youth is not a time of life; it is a state of mind; it is not a matter of rosy cheeks, red lips and supple knees; it is a matter of the will, a quality of the imagination, a vigor of the emotions; it is the freshness of the deep springs of life. &lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>30</y>
        <width>431</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Press &quot;Record&quot; and read the following text as clearly as you can.</string>
      </property>
     </widget>
     <widget class="QLabel" name="Animation">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>160</y>
        <width>61</width>
        <height>61</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QWidget" name="horizontalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>230</y>
        <width>411</width>
        <height>61</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="enroll">
         <property name="text">
          <string>Enroll!</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="startTrain">
         <property name="text">
          <string>Train!</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="dumpBtn">
         <property name="text">
          <string>Dump</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="loadBtn">
         <property name="text">
          <string>Load</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="horizontalLayoutWidget_2">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>180</y>
        <width>221</width>
        <height>41</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QPushButton" name="enrollRecord">
         <property name="text">
          <string>Record</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="stopEnrollRecord">
         <property name="text">
          <string>Stop</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QLabel" name="enrollTime">
      <property name="geometry">
       <rect>
        <x>73</x>
        <y>190</y>
        <width>51</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>00:00</string>
      </property>
     </widget>
     <widget class="QToolButton" name="enrollFile">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>185</y>
        <width>31</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>...</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="Recognition">
    <attribute name="title">
     <string>Recognition</string>
    </attribute>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>0</y>
       <width>561</width>
       <height>341</height>
      </rect>
     </property>
     <property name="title">
      <string>Recognition</string>
     </property>
     <widget class="QWidget" name="horizontalLayoutWidget_3">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>30</y>
        <width>271</width>
        <height>41</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <widget class="QCheckBox" name="Reco_Verificationcheckbox">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string>Claim identity:</string>
         </property>
         <property name="iconSize">
          <size>
           <width>24</width>
           <height>24</height>
          </size>
         </property>
         <property name="tristate">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="Reco_Verification">
         <property name="enabled">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QLabel" name="Animation_2">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>70</y>
        <width>61</width>
        <height>61</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QWidget" name="verticalLayoutWidget_3">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>180</y>
        <width>131</width>
        <height>161</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLabel" name="Reco_Userimage">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="recoUsername">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Unknown</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QLabel" name="recoTime">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>100</y>
        <width>51</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>00:00</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>62</y>
        <width>201</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="text">
       <string>* Tick and we will verify your identity</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>180</y>
        <width>231</width>
        <height>151</height>
       </rect>
      </property>
      <property name="text">
       <string>Here is a little animation</string>
      </property>
     </widget>
     <widget class="QPushButton" name="newReco">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>35</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>New Task</string>
      </property>
     </widget>
     <widget class="QProgressBar" name="recoProgressBar">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>140</y>
        <width>401</width>
        <height>23</height>
       </rect>
      </property>
      <property name="value">
       <number>0</number>
      </property>
     </widget>
     <widget class="QWidget" name="horizontalLayoutWidget_6">
      <property name="geometry">
       <rect>
        <x>182</x>
        <y>90</y>
        <width>191</width>
        <height>41</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_9">
       <item>
        <widget class="QPushButton" name="recoRecord">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Record</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="stopRecoRecord">
         <property name="text">
          <string>Stop</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QToolButton" name="recoFile">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>95</y>
        <width>29</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>...</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_6">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>330</y>
       <width>411</width>
       <height>171</height>
      </rect>
     </property>
     <property name="title">
      <string>Multiple tasks from files</string>
     </property>
     <widget class="QWidget" name="horizontalLayoutWidget_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>401</width>
        <height>41</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_7">
       <item>
        <widget class="QLabel" name="label_8">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Input folder:   </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="Reco_Inputfile"/>
       </item>
       <item>
        <widget class="QToolButton" name="Reco_Inputchooser">
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="horizontalLayoutWidget_5">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>80</y>
        <width>401</width>
        <height>41</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_8">
       <item>
        <widget class="QLabel" name="label_9">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Output folder:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="Reco_Outputfile"/>
       </item>
       <item>
        <widget class="QToolButton" name="Reco_Outputchooser">
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QProgressBar" name="Reco_progressBar">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>130</y>
        <width>261</width>
        <height>31</height>
       </rect>
      </property>
      <property name="value">
       <number>0</number>
      </property>
     </widget>
     <widget class="QPushButton" name="Reco_Start">
      <property name="geometry">
       <rect>
        <x>315</x>
        <y>130</y>
        <width>96</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Start</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="Conversation">
    <attribute name="title">
     <string>Conversation Mode</string>
    </attribute>
    <widget class="QGroupBox" name="groupBox_4">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>30</y>
       <width>201</width>
       <height>431</height>
      </rect>
     </property>
     <property name="title">
      <string>Conversation Info</string>
     </property>
     <widget class="QTextBrowser" name="Conv_Info">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>30</y>
        <width>193</width>
        <height>351</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="Conv_Saveas">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>390</y>
        <width>96</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Save as..</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Conv_Clear">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>390</y>
        <width>96</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Clear</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_5">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>180</y>
       <width>141</width>
       <height>211</height>
      </rect>
     </property>
     <property name="title">
      <string>Current Speaker</string>
     </property>
     <widget class="QLineEdit" name="convUsername">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>170</y>
        <width>131</width>
        <height>33</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
     <widget class="QLabel" name="Conv_Userimage">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>40</y>
        <width>129</width>
        <height>112</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QPushButton" name="Conv_Startrecord">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>60</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Record</string>
     </property>
    </widget>
    <widget class="QLabel" name="convTime">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>100</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>00:00</string>
     </property>
    </widget>
    <widget class="QLabel" name="Animation_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>61</width>
       <height>61</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="Video">
    <attribute name="title">
     <string>Video Mode</string>
    </attribute>
    <widget class="QMdiArea" name="mdiArea">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>110</y>
       <width>301</width>
       <height>201</height>
      </rect>
     </property>
    </widget>
    <widget class="QSlider" name="horizontalSlider">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>310</y>
       <width>301</width>
       <height>51</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QPushButton" name="OpenVideo">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>60</y>
       <width>96</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Open...</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>190</y>
       <width>261</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>This page is under construction</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>Reco_Verificationcheckbox</sender>
   <signal>toggled(bool)</signal>
   <receiver>Reco_Verification</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>186</x>
     <y>108</y>
    </hint>
    <hint type="destinationlabel">
     <x>343</x>
     <y>107</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
